exporters/FusionRobotExporter/Source/Data/BXDJ/Driver.cpp

```diff
@@ -127,39 +128,46 @@ nlohmann::json BXDJ::Driver::GetExportJson()
 	return exportJson;
 }
 
-void Driver::loadJSONObject(const rapidjson::Value & driverJSON)
+void Driver::loadJSONObject(const nlohmann::json driverJSON)
 {
-	if (driverJSON.IsObject())
+	if (driverJSON.is_object())
 	{
 		// Driver Properties
-		if (driverJSON.HasMember("type") && driverJSON["type"].IsNumber())
-			type = (Driver::Type)driverJSON["type"].GetInt();
-		if (driverJSON.HasMember("signal") && driverJSON["signal"].IsNumber())
-			portSignal = (Driver::Signal)driverJSON["signal"].GetInt();
-		if (driverJSON.HasMember("portOne") && driverJSON["portOne"].IsNumber())
-			portOne = driverJSON["portOne"].GetInt();
-		if (driverJSON.HasMember("portTwo") && driverJSON["portTwo"].IsNumber())
-			portTwo = driverJSON["portTwo"].GetInt();
 
-		// Components
+		if (driverJSON["type"].is_number()) {
+			type = (Driver::Type)driverJSON["type"].get<int>();
+		}
 
-		if (driverJSON.HasMember("wheel") && driverJSON["wheel"].IsObject())
-		{
-			const rapidjson::Value& wheelJSON = driverJSON["wheel"];
+		if (driverJSON["signal"].is_number()) {
+			portSignal = (Driver::Signal)driverJSON["signal"].get<int>();
+		}
+
+		if (driverJSON["portOne"].is_number()) {
+			portOne = driverJSON["portOne"].get<int>();
+		}
+
+		if (driverJSON["portTwo"].is_number()) {
+			portTwo = driverJSON["portTwo"].get<int>();
+		}
+
```

Comment:
the `inputGear` and `outputGear` fields should also be assigned here.

Reply:
This is handled in the wheel json

Reply:
Aren't all joint drivers supposed to have an `inputGear` and `outputGear`?

Reply:
Correct. Fixed.